src/qml/mixxxscreen.cpp

```diff
@@ -3,6 +3,37 @@
 namespace mixxx {
 namespace qml {
 
+void MixxxScreen::classBegin() {
+}
+
+void MixxxScreen::componentComplete() {
+    const auto* const meta = metaObject();
+    int typedIdx = meta->indexOfMethod(kScreenTransformFunctionTypedSignature);
+    int untypedIdx = meta->indexOfMethod(kScreenTransformFunctionUntypedSignature);
+    if (typedIdx >= 0) {
+        auto transformMethod = meta->method(typedIdx);
+        if (!transformMethod.isValid()) {
+            // TODO
+        } else {
+            transform = [transformMethod, this](const QByteArray input,
+                                const QDateTime& timestamp) -> QVariant {
+                return transform(transformMethod, input, timestamp, true);
+            };
```

Comment:
This should be a bit more efficient than previous implementation since transform function is searched only once trhough relfection. But then again, I know very few C++ so I may awefully wrong.